REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 1.1
```

Comment:
I would not require versions > v1.0. Many people will use that version. You even "highly recommended" that recently and the text in `README` says v1.0 :wink:
(For the same reason, do not drop the version in Travis.)

Reply:
By the way, I think the `REQUIRE` file is redundant if there is a `Project` file...

Reply:
Yes, that's true. Should i delete the REQUIRE then?

Reply:
If it works, yes!

Reply:
But wait! The `Project` file does not restrict the Julia version.

Reply:
Yes, i was thinking about that. Seems like [it has to be specified under the [compat] tag](https://julialang.github.io/Pkg.jl/v1/compatibility/#**6.**-Compatibility-1) as in: 

```julia
[compat]
julia = "1.0"
```